Yoast/wordpress-seo
# Prevent huge error logs

## Summary


This PR can be summarized in the following changelog entry:

* Fixes an issue where huge log files could be generated if PHP was configured to also log the arguments in debug backtraces.

## Test instructions

This PR can be tested by following these steps:

To check logs will remain small even if output:
* @Djennez could you test this one? I've been trying to replicate the exact logs you're getting where only arguments are getting logged but have had no success thus far.

To check logs aren't output at all when not needed:
* Add `do_action( 'wpseo_head' );` to the top of your `functions.php`.
* There should be no notices output.

## UI changes

## Documentation

## Quality assurance


Fixes #15058.
